demisto_sdk/commands/validate/validators/RN_validators/RN114_validate_release_notes_header.py

```diff
@@ -0,0 +1,210 @@
+from __future__ import annotations
+
+import re
+from typing import Dict, Iterable, List, Set, Tuple
+
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.content_graph.common import (
+    ContentType,
+)
+from demisto_sdk.commands.content_graph.objects.content_item import ContentItem
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+CONTENT_TYPE_SECTION = re.compile(r"^#### ([\w ]+)$\n([\w\W]*?)(?=^#### |\Z)", re.M)
+CONTENT_ITEM_SECTION = re.compile(
+    r"^##### (.+)$\n([\w\W]*?)(?=^##### |\Z)|^- (?:New: )?$", re.M
+)
+
+
+class ReleaseNoteHeaderValidator(BaseValidator[ContentTypes]):
+    error_code = "RN114"
+    description = "Validate the existence of content types in the first-level headers (####) and the content items in second-level headers (#####)."
+    rationale = (
+        "Providing documentation with accurate information and avoiding confusion."
+    )
+    error_message = (
+        "The following release note headers are invalid:\n"
+        "{content_type_message}\n{content_item_message}\n"
+    )
+    related_field = "release_note"
+    is_auto_fixable = False
+    related_file_type = [RelatedFileType.RELEASE_NOTE]
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        validator_results: List[ValidationResult] = []
+        for content_item in content_items:
+            if content_item.pack_metadata_dict and content_item.pack_metadata_dict.get(
+                "hidden"
+            ):
+                logger.debug("Content item is marked as hidden. Skipping.")
+                continue
+            (
+                invalid_headers_content_type,
+                invalid_headers_content_item,
+            ) = self.validate_convert_content_type_to_rn_headers(content_item)
```

Comment:
```suggestion
            ) = self.validate_release_notes_headers(content_item)
```
the previous one was ok :)